### ISSUE TITLE
cannot create eosio. bpay 

### ISSUE THREAD

**Issue by yoo3740** [EOSIO/eos#5119]
`nodeos -e -p eosio --private-key '["EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV","5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3"]' --plugin eosio::producer_plugin --plugin eosio::chain_api_plugin --plugin eosio::http_plugin --plugin eosio::history_api_plugin  `  

AND

cleos wallet import 

private key: imported private key for: 5HwpAUTwcRJUA93dsxc2NiCtUJDRvyWzFGRMYC5iW4oDQX5LHJW 

cleos create account eosio eosio.bpay EOS5wTQVBtuiz4wCH1dXBGdv2a2y7Y78MHomZGgAw9ASW6kE9z8zN 

 BUT there is any respond
![image](https://user-images.githubusercontent.com/22166187/43875034-f55cb210-9bc9-11e8-9edf-68ce553dfbb3.png)

**Comment by andriantolie** [comment]
Try to start your nodeos with `--verbose-http-errors` to see a more detailed error message. Nevertheless, it should still show the brief error message even without the `--verbose-http-errors` arg, so the fact that nothing appears seems weird.

Does it work fine when you try to create other accounts? Creating eosio.bpay account should have no difference with creating another account.

**Comment by yoo3740** [comment]
@andriantolie i did with` --verbose-http-errors ` but same. and couldn't create other accounts too.
but when I stop the nodeos, an error message show in the cleos command. `Failed to connect to nodeos at http://127.0.0.1:8888/; is nodeos running?
`

**Comment by ashishchandr70** [comment]
I had the same issue. I then checked for how many instances of **keosd** were running and found that there were 2: one on port 8900 and another one on 8899 (which the tutorial had us create). I killed nodeos, keosd and then restarted nodeos. 

Then, I created the account with the cleos create account command and it worked. 

![image](https://user-images.githubusercontent.com/41921015/44234179-96e59100-a16b-11e8-91b6-d72893d77fb8.png)

Hope this helps.
